### ISSUE TITLE
Accordion panel content is not appropriately hidden

### ISSUE THREAD

**Issue by scottaohara** [stuartjnelson/badger-accordion#16]
# Expected Behavior

When accordion panels are collapsed, they should be hidden to all users.

# Current Behavior & failure information

Presently when accordion panels are collapsed, screen reader users can access all of the contents of the collapsed accordions.  Additionally, if there are focusable elements within an accordion panel, a sighted keyboard user navigating by the tab key would lose their focus on these elements.

## Steps to Reproduce

1. Turn on a screen reader
2. begin navigating the page with the virtual cursor
3. content that should be hidden is announced to the screen reader

or

1. Add a focusable element/s to an accordion panel
2. Navigate the page with the tab key
3. focus will become "lost" when navigating from one accordion trigger to the next, as the hidden element(s) have received keyboard focus.

# Fix
It's a rather simple fix to correct this.  Add `visibility: hidden` to the CSS for the `-ba-is-hidden` class and when this class is on the containing element, the contents inside will be properly hidden to all users.
